workbenchfx-core/src/main/java/com/dlsc/workbenchfx/view/ToolbarPresenter.java

```diff
@@ -51,6 +52,11 @@ public ToolbarPresenter(Workbench model, ToolbarView view) {
     init();
     // Adds initially a menuButton if necessary (size of items > 0)
     setupMenuBtn();
+
+    // check initially whether to use the single module layout (size of modules = 1)
+    if (model.getModules().size() == 1) {
```

Comment:
To make intention clearer and reduce duplicated code, we could refactor this check into a method on the model `isSingleModuleLayout()`, which we could also use it below in the listener and in the `WorkbenchSkin`

Reply:
Don't forget to refractor this check as well! 😉 Afterwards it should be good to go!

Reply:
Oops, I missed that. Thanks for pointing it out. Fixed it.